src/parser/DASHTree.cpp

```diff
@@ -1052,6 +1052,10 @@ static void XMLCALL start(void* data, const char* el, const char** attr)
       {
         uint64_t dur(0);
         AddDuration((const char*)*(attr + 1), dur, 1500);
+        // 0S minimumUpdatePeriod = refresh after every segment
+        // We already do that so lets set our minimum updateInterval to 30s
+        if (dur == 0)
+          dur = 30000;
```

Comment:
The change looks fine to me but why is the duration above add with a scale of 1500?

Reply:
Haha. I've prd what I thought was fix before for that. Glenn explains why here:
https://github.com/xbmc/inputstream.adaptive/pull/658#issuecomment-825609131

Reply:
Ha, ok. Cool. Makes sense. I would argue that it should have a comment explaining that use a wrapper function with a different name!